OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1010,6 +1010,68 @@ public static DreamValue NativeProc_hascall(NativeProc.Bundle bundle, DreamObjec
         return new DreamValue(obj.ObjectDefinition.HasProc(procName) ? 1 : 0);
     }
 
+    [DreamProc("hearers")]
+    [DreamProcParameter("Depth", Type = DreamValueTypeFlag.Float)]
+    [DreamProcParameter("Center", Type = DreamValueTypeFlag.DreamObject)]
+    public static DreamValue NativeProc_hearers(NativeProc.Bundle bundle, DreamObject? src, DreamObject? usr) { //TODO: Change depending on center
```

Comment:
Rather than copy-pasting all of `viewers()`, this code should be moved into a method `DreamProcNativeHelpers.cs` that can be called by both native procs.